raspberry/read_sbus/radioDataParser.py

```diff
@@ -0,0 +1,258 @@
+
+    def armed(self):
+        """Checks if the arm switch (switch B) has been toggled.
+
+        :return: Arm status. True if armed else False.
+        :rtype: bool
+        """
+        return self.m_channelData[RADIO_CHANNEL_SWITCH_B] >= 1500
+
+    def kill(self):
+        """Checks if the kill switch (switch A) has been toggled.
+
+        :return: Kill status. True if killed else False.
+        :rtype: bool
+        """
+        return self.m_channelData[RADIO_CHANNEL_SWITCH_A] >= 1500
+
+    def switchC(self):
+        """Checks the position of the three-way switch C.
+
+        :return: Three-way switch position. DOWN = 0, MID = 1, UP = 2.
+        :rtype: Of class ThreeWaySwitch, extends Enum
+        """
+        self.switchValue = self.m_channelData[RADIO_CHANNEL_SWITCH_C]
+        if self.switchValue <= 450:
+            return ThreeWaySwitch.DOWN
+        elif self.switchValue <= 1200:
+            return ThreeWaySwitch.MID
+        return ThreeWaySwitch.UP
+
+    def switchD(self):
+        """Checks the positon of the switch D.
+
+        :return: Switch D position.
+        :rtype: bool
+        """
+        return self.m_channelData[RADIO_CHANNEL_SWITCH_D] >= 1500
+
+    def trimmerVRAPercentage(self):
+        """Maps Variable Resistor A data to percentage.
+
+        :return: VRA percentage in range [0, 1].
+        :rtype: float
+        """
+        return self.mapTrimmerToPercentage(self.m_channelData[RADIO_CHANNEL_VRA])
+
+    def trimmerVRCPercentage(self):
+        """Maps Variable Resistor C data to percentage.
+
+        :return: VRC percentage in range [0, 1].
+        :rtype: float
+        """
+        return self.mapTrimmerToPercentage(self.m_channelData[RADIO_CHANNEL_VRC])
+
+    def trimmerVRBPercentage(self):
+        """Maps Variable Resistor B data to percentage.
+
+        :return: VRB percentage in range [0, 1].
+        :rtype: float
+        """
+        return self.mapTrimmerToPercentage(self.m_channelData[RADIO_CHANNEL_VRB])
+
+    def trimmerVREPercentage(self):
+        """Maps Variable Resistor E data to percentage.
+
+        :return: VRE percentage in range [0, 1].
+        :rtype: float
+        """
+        return self.mapTrimmerToPercentage(self.m_channelData[RADIO_CHANNEL_VRE])
+    
+    def mapPrcnt(self, percentage, minVal, maxVal):
+        """Linearly maps percentage data to range [minVal, maxVal].
+
+        :param percentage: Value of percentage, in range [0, 1], to be mapped.
+        :type percentage: float
+        :param minVal: Lower boundary of the mapped data.
+        :type minVal: int/ float
+        :param maxVal: Upper boundary of the mapped data.
+        :type maxVal: int/ float
+        :return: mapped data in range [minVal, maxVal].
+        :rtype: int/ float
+        """
+        return minVal + percentage * (maxVal - minVal)
+    
+    def formatRadioDataForSending(self):
+        """Processes, encodes, and formats the radio data into a 
```

Comment:
@gunturiCM I updated this docstring. Docstrings should be formatted using markdown.

Reply:
I will keep this in mind.